src/signin/signinActions.js

```diff
@@ -33,18 +33,17 @@ export function toSignin() {
   };
 }
 
-export function signin(name, password) {
+export function signin(username, password) {
```

Comment:
Actually, this would just destructure value then create an object again with it when you stringify it. Just pass in value and stringify that.